person.rb

```diff
@@ -2,8 +2,8 @@
 require './rental'
 
 class Person < Nameable
-  attr_reader :id, :rentals
-  attr_accessor :name, :age
+  attr_reader :id
+  attr_accessor :name, :age, :rentals
```

Comment:
Addressed

Reply:
Hello @DeliceLydia,
I have mentioned submitting the exercise that those points were misinterpreted and need a re-review. The reason is that I am passing the Rentals object inside the Rentals class to the Book and Person class so the object does not need to be created in any manner.